contents/blog/rome-hackathon.md

```diff
@@ -0,0 +1,82 @@
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/planning-a-company-offsite/planning-offsite.jpeg
+featuredImageType: full
+categories: ["Inside PostHog", "Startups"]
+---
+
+As an all-remote team, we know how important getting together in person is (we’ve [written about this before](/blog/asynchronous-remote-companies)). Our [product analytics team](/handbook/people/team-structure/product-analytics) (plus Luke, Annika, Cory, and Lottie) recently got back from an offsite in Rome – see also [how we plan a killer offsite in 8 weeks](/blog/planning-a-company-offsite). 
+
+The team ate a lot of pizza and gelato, refocused and aligned current work, ran 360 feedback sessions, and more (see our engineer Paul's [summary thread here](https://twitter.com/pauldambra/status/1576257069766037506)). On top of all of this, they had a hackathon and built a bunch of new things.
+
+![Team Rome](../images/blog/rome-hackathon/team-rome.jpeg)
+
+We are showcasing them here. Some are sneak previews, others are works-in-progress, and they’ll likely change. We wanted to give you an idea of what our team actually does at an offsite hackathon, so here's what they built.
+
+## Frontend injected apps
+
+Have you ever wanted to add floating pineapples to every page on your site? Marius worked on a way to make this dream a reality. More specifically, he built the ability to inject code into your page through [posthog-js](https://posthog.com/docs/integrate/client/js).
+
+![Pineapple mode](../images/blog/rome-hackathon/pineapple-mode.gif)
+
+To do so, PostHog app developers add a `web.ts` file with the code they want to inject through PostHog into their app. This enables them to add features like notification banners and custom forms (or pineapples). 
+
+Read more details in the [PR here](https://github.com/PostHog/posthog/pull/12003).
+
+## Feedback app
+
+As a more serious example of what the frontend injected apps look like, Marius and Luke built a feedback app.
+
+The feedback app is injected into your site through PostHog. It can be modified quickly by changing the `web.ts` file, without having to ship changes to your site. On your site, it takes input from users and delivers responses to their own tab in PostHog.
+
+![Feedback app](../images/blog/rome-hackathon/feedback-app.png)
+
+You can find the first PR for the feedback app [here](https://github.com/PostHog/feedback-app/pull/1).
+
+## FaceHog (HogBook? Hoggr?) notifications
+
+A problem some PostHog users face is others changing their carefully crafted insights and feature flags. They can mess these up or completely ruin them, and the original creator wouldn’t have any idea. Worse still, users could delete them without notice.
+
+To limit this happening, Paul worked on “FaceHog,” a notification log of changes to insights, dashboards, feature flags, and experiments you have created.
```

Comment:
Is the comma in `“FaceHog,”` expected?

Reply:
Yeah, I like to keep my punctuation within the quotes, but I realize it looks particularly awkward here.